### DISCUSSION TITLE
哪个Arduino-ESP32版本有穩定的SD_MMC库？

### DISCUSSION THREAD

**Discussion by ham0nd** [moononournation/Arduino_GFX#239] [Q&A]
在瀏覽https://www.instructables.com/Mini-Retro-TV/過程中，老師提到 `with latest Arduino-ESP32 version, the SD_MMC library become very unstable and cannot compatible with most SD card.` 
![image](https://user-images.githubusercontent.com/43801159/208573180-6d3436ff-83e2-4cb8-86e0-d8000634c935.png)

所以我想瞭解一下以前哪个Arduino-ESP32版本具有穩定的SD_MMC库？

**Comment by moononournation** [comment]
may be try 1.x version
